subprojects/packagefiles/cpputest/src/Platforms/meson.build

```diff
@@ -3,7 +3,7 @@ cpputest_platform = get_option('cpputest_platform')
 if cpputest_platform != 'none'
   if cpputest_platform == 'autodetect'
     compiler_id = meson.get_compiler('cpp').get_id()
-    if compiler_id == 'clang' or compiler_id == 'gcc'
+    if compiler_id == 'gcc' or compiler_id == 'clang' or compiler_id == 'clang-cl'
```

Comment:
should it be treated as GCC or VisualCpp?

Reply:
Yeah, it passes here so I'm not sure it's just a funky compatibility layer, or intended behaviour.
I'll throw it to VisualCpp and let's see what happens

Reply:
After quick read, yeah, this should definitely be treated as VisualCpp